pilot/pkg/config/kube/gateway/model.go

```diff
@@ -33,6 +33,7 @@ const (
 	gatewaySAOverride            = "gateway.istio.io/service-account"
 	serviceTypeOverride          = "networking.istio.io/service-type"
 	addressTypeOverride          = "networking.istio.io/address-type"
+	ipFamilyPolicyOverride       = "networking.istio.io/ip-family-policy"
```

Comment:
This is not an Istio field/API and is only for gateways - can we make that clearer in the label scoping?